lib/raidenclient/RaidenClient.ts

```diff
@@ -134,7 +157,24 @@ class RaidenClient extends BaseClient {
     }
   }
 
-  public sendPayment = async (_deal: SwapDeal): Promise<string> => {
+  public sendPayment = async (deal: SwapDeal): Promise<string> => {
+    assert(deal.state === SwapState.Active);
+    assert(deal.destination);
+    let amount = 0;
+    let tokenAddress = '';
+    if (deal.role === SwapRole.Maker) {
+      // we are the maker paying the taker
+      amount = deal.makerAmount;
```

Comment:
I think this should be `takerAmount` as that's the amount the taker is expecting to receive.